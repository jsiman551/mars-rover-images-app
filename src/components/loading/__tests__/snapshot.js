import { render } from '@testing-library/react';
import Loading from '../index';

it('renders loading unchanged', () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});
