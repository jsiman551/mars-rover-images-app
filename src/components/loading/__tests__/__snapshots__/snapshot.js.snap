// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading unchanged 1`] = `
<div>
  <div
    class="css-xi606m"
    data-testid="loading-container"
  >
    <div
      class="chakra-spinner css-u4uvb9"
      data-testid="loading-spinner"
    >
      <span
        class="css-8b45rq"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
